crates/astria-core/Cargo.toml

```diff
@@ -19,8 +19,9 @@ keywords = ["astria", "grpc", "rpc", "blockchain", "execution", "protobuf"]
 bech32 = "0.11.0"
 brotli = { version = "5.0.0", optional = true }
 celestia-types = { version = "0.1.1", optional = true }
-pbjson = { version = "0.6.0", optional = true }
+pbjson = { version = "0.6.0", git = "https://github.com/astriaorg/pbjson.git", branch = "url-safe", optional = true }
```

Comment:
Looks like we can't bump to `0.7.0` yet, because we also need `pbjson-types`, which itself requires `prost 0.12`. :-/